pages/cart.html

```diff
@@ -0,0 +1,118 @@
+                        <span>Quantity: </span>
+                        <div class = "quantiy-section">
+                            <button>-</button>
+                            <input type = "number" value = "1"/>
+                            <button>+</button>
+                        </div>
+                    <button class="btn-addtocart border-radius-5px"></i>Move To Wishlist</button>
+                    <button class="btn-addtocart border-radius-5px btn-remove"></i>Remove From Cart</button>
+                </div>
+            </div>
+            <div class="basic-card basic-card-horizontal border-radius-5px">
+                <div class = "img-section">
+                    <img class ="card-img card-flex-item border-radius-5px" src="/assets/heart-shape-mould.jpeg">
+                    <span class="badge-on-card">New</span>
+                </div>
+                <div class="card-body">
+                    <p>Trendy Elegant Women Tops & Tunics</p>
+                    <div class = "price-details">
+                        <span class = "new-price">&#x20B9 210</span>
+                        <span class = "old-price">&#x20B9 299</span>
+                        <span class = "offer">30% off</spam>
+                    </div>
+                        <span>Quantity: </span>
+                        <div class = "quantiy-section">
+                            <button>-</button>
+                            <input type = "number" value = "1"/>
+                            <button>+</button>
+                        </div>
+                    <button class="btn-addtocart border-radius-5px"></i>Move To Wishlist</button>
+                    <button class="btn-addtocart border-radius-5px btn-remove"></i>Remove From Cart</button>
+                </div>
+            </div>
+        </div>
+
+        <div class = "price-details-card">
+                <h3>Price Details</h3>
+                <div class = "price-division">
+                    <div class = "price-division-item price-detail-item">
+                        <span>Price (2 items)</span>
+                        <span>&#x20B9 420</span>
+                    </div>
+                    <div class = "price-division-item price-detail-item">
+                        <span>Discount</span>
+                        <span>- &#x20B9 168</span>
+                    </div>
+                    <div class = "price-division-item price-detail-item">
+                        <span>Delivery Charges</span>
+                        <span>&#x20B9 49</span>
+                    </div>
+                </div>
+                
+                <div class = "price-division-item price-detail-item">
+                    <span>Total Amount</span>
+                    <span>&#x20B9 301</span>
+                </div>
+            <div class = "save-message price-detail-item">You will save &#x20B9 168 on this order</div>
+            <button class="btn-order"> PLACE ORDER</button>
+
```

Comment:
You can remove unwanted space

Reply:
Thanks for pointing that out. Removed extra spaces.